kedro/templates/project/{{ cookiecutter.repo_name }}/tests/test_run.py

```diff
@@ -1,42 +1,20 @@
 """
-This module contains an example test.
-
+This module contains example tests for a Kedro project.
 Tests should be placed in ``src/tests``, in modules that mirror your
-project's structure, and in files named test_*.py. They are simply functions
-named ``test_*`` which test a unit of logic.
-
-To run the tests, run ``pytest`` from the project root directory.
+project's structure, and in files named test_*.py.
 """
-
 from pathlib import Path
 
-import pytest
-
-from kedro.config import OmegaConfigLoader
-from kedro.framework.context import KedroContext
-from kedro.framework.hooks import _create_hook_manager
-from kedro.framework.project import settings
-
-
-@pytest.fixture
-def config_loader():
-    return OmegaConfigLoader(conf_source=str(Path.cwd() / settings.CONF_SOURCE))
-
-
-@pytest.fixture
-def project_context(config_loader):
-    return KedroContext(
-        package_name="{{ cookiecutter.python_package }}",
-        project_path=Path.cwd(),
-        env="local",
-        config_loader=config_loader,
-        hook_manager=_create_hook_manager(),
-    )
-
+from kedro.framework.session import KedroSession
+from kedro.framework.startup import bootstrap_project
 
 # The tests below are here for the demonstration purpose
 # and should be replaced with the ones testing the project
 # functionality
-class TestProjectContext:
-    def test_project_path(self, project_context):
-        assert project_context.project_path == Path.cwd()
+
+class TestKedroRun:
+    def test_kedro_run(self):
+        bootstrap_project(Path.cwd())
+
+        with KedroSession.create(project_path=Path.cwd()) as session:
```

Comment:
If you do this, how do we test hooks?

Reply:
We were discussing [here](https://github.com/kedro-org/kedro-starters/pull/261#pullrequestreview-2478629238) that the test should be focused on executing a run. Maybe we could have another example for hook functionality?

Reply:
So I wasn't part of that discussion - but I was part of the design of the initial test file design and this was the reason we use a context not session. This is simpler for most use-cases so we can just put a hook testing part in the docs, but also it would be nice to just pass hooks to the session.

Reply:
That's good to know, thanks for the context! I can change the test to pass the hooks to the session.